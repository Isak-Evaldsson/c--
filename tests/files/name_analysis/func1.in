int fail(int a) {
    return b;
}

int succ(int a) {
    return a;
}