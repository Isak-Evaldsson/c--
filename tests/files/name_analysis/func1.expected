104.12: Invalid symbol 'b'
